config.py

```diff
@@ -49,4 +49,10 @@
 ONTOLOGY_API_URL = os.environ.get('ONTOLOGY_API_URL', 'https://ontology.staging.archive.data.humancellatlas.org')
 
 DIRECT_SUBMISSION = os.environ.get('DIRECT_SUBMISSION', False)
-BIOSAMPLES_URL = os.environ.get('BIOSAMPLES_URL', 'https://wwwdev.ebi.ac.uk/biosamples')
\ No newline at end of file
+BIOSAMPLES_URL = os.environ.get('BIOSAMPLES_URL', 'https://wwwdev.ebi.ac.uk/biosamples')
+
+BIOSTUDIES_URL = os.environ.get('BIOSAMPLES_URL', 'http://biostudy-bia.ebi.ac.uk:8788')
+BIOSTUDIES_USERNAME = 'unifieds'
+BIOSTUDIES_USERNAME = os.environ.get('BIOSTUDIES_USERNAME', BIOSTUDIES_USERNAME)
+BIOSTUDIES_PASSWORD = 'INSERT_PASSWORD'
+BIOSTUDIES_PASSWORD = os.environ.get('BIOSTUDIES_PASSWORD', BIOSTUDIES_PASSWORD)
```

Comment:
This is probably patterned with the initialisation of AAP_API_PASSWORD, however i think it's better to not do defaulting for this env variable and just make it required. You could just let it error when that env variable is not found. I also worry that a developer testing/running this locally would try to write the password directly to config file and would make it prone to be committed to the repo.

Reply:
@aaclan-ebi OK, but then it is true for all the similar variables, right? I shall change al of then.

Reply:
Done and pushed

Reply:
Thanks!

Reply:
actually doing just `os.environ['BIOSTUDIES_PASSWORD']` is actually better because it would already throw an error 😅